kmprevenuecat-purchases/src/iosMain/kotlin/com/mmk/kmprevenuecat/purchases/Mapper.ios.kt

```diff
@@ -100,7 +100,8 @@ internal fun RCEntitlementInfo.asEntitlementInfo(): EntitlementInfo {
 public fun RCCustomerInfo.asCustomerInfo(): CustomerInfo {
     return CustomerInfo(
         originalAppUserId = originalAppUserId(),
-        entitlements = entitlements().asEntitlementInfos()
+        entitlements = entitlements().asEntitlementInfos(),
+        managementURL = managementURL().toString()
```

Comment:
I am not sure if `absoluteString` and `toString` for NSUrl is the same.

Reply:
Replaced `toString` with `absoluteString`

Reply:
Nice! Now that you mention it, maybe I was a bit too fast when implementing this to be a String in `CustomerInfo`. Might actually be nice to add a wrapper class that could return the url as a String and have `nativeUrl` property exposing the original Uri on Android and the original NSURL on iOS.

[Firebase-kotlin-sdk](https://github.com/GitLiveApp/firebase-kotlin-sdk/blob/master/firebase-firestore/src/iosMain/kotlin/dev/gitlive/firebase/firestore/firestore.kt#L322) does this.